docs/ORGANIZATION.md

```diff
@@ -1,21 +1,108 @@
 
-Tutorials, the main index page, and the WIP sphinx theme are in the [`napari/napari.github.io`] repo.
+Tutorials, the main index page, and the napari sphinx theme are in the
+[`napari/napari.github.io`] repo.
 
 The [`napari/docs`] repo contains no original documentation sources.
 
 ## Bringing it all together
 
-For the [`napari/docs`] repo, files are built in `docs/_build/html` in the main [`napari/napari`] repo and then copied over to the appropriate version number or `dev` using an automated process.
+For the [`napari/docs`] repo, files are built in `docs/_build/html` in the main
+[`napari/napari`] repo and then copied over to the appropriate version number or
+`dev` using an automated process.
 
-Documentation sources are copied over to `napari.github.io` using the `copy-docs.py` script (found in the `docs` directory of [`napari/napari`]). Duplicate files are excluded and the table of contents is automatically updated. Specifying which files should be copied / excluded can be modified in the script itself. This script is automatically run in continuous integration and the changes are pushed to the [`napari/napari.github.io`] repo, which will then automatically build and update the site.
+Documentation sources are copied over to `napari.github.io` using the
+`copy-docs.py` script (found in the `docs` directory of [`napari/napari`]).
+Duplicate files are excluded and the table of contents is automatically updated.
+Specifying which files should be copied / excluded can be modified in the script
+itself. This script is automatically run in continuous integration and the
+changes are pushed to the [`napari/napari.github.io`] repo, which will then
+automatically build and update the site.
+
+## Making changes to the documentation
+
+For any changes in documentation (except the tutorials), a PR should be
+submitted to the [`napari/napari`] repo. When a new file is added to this repo
+under the current folder structure, it will automatically be copied over to
+[`napari/napari.github.io`] and the [napari.org] table of contents will be
+automatically updated. If the changes are done directly on
+[`napari/napari.github.io`], they will be overwritten when a new PR is merged to
+[`napari/napari`].
+
+If any autogenerated documentation sources are added to [`napari/napari`], they
+will need to also be added to [`napari/napari.github.io`] and to the list of
+documentation sources to be copied over by the `copy-docs.py` script.
+
+## Building and testing documentation and the napari.org website locally
+
+If you want to check that your documentation changes are rendered correctly,
+you will need to check them usign the theme from the [`napari/napari.github.io`]
+repository.
+
+### 1. Clone both repos
+
+In order to see the changes exactly as they will appear in [napari.org], you
+need to clone both the [`napari/napari`] and the [`napari/napari.github.io`]
+repos.
+
```

Comment:
You may wan to be more explicit and say they they should be cloned next to each other and that the followign structure is expected:

```
.
├── napari. -> clone of napari/napari
│   └── docs
│       └── copy-docs.py
└── napari.github.io -> clone of napari.github.io
```